### ISSUE TITLE
Excuse me.I can't find the function “get_RT_bat”  in utils_funs.py when I run "train_G2L.py".

### ISSUE THREAD

**Issue by GuanghuXie** [DC1991/G2L_Net#15]

**Comment by DC1991** [comment]
Sorry, I maybe delete that code accidentally. The get_RT_bat is defined as below:
def get_RT_bat(P,Q):

    '''
    :param P: B*N*3
    :param Q: B*N*3
    :return:
    '''

    P1=P-P.mean(dim=1,keepdim=True)
    Q1=Q-Q.mean(dim=1, keepdim=True)

    C = torch.bmm(P1.transpose(1,2), Q1)## B*3*3

    # Computation of the optimal rotation matrix
    # This can be done using singular value decomposition (SVD)
    # Getting the sign of the det(V)*(W) to decide
    # whether we need to correct our rotation matrix to ensure a
    # right-handed coordinate system.
    # And finally calculating the optimal rotation matrix U
    # see http://en.wikipedia.org/wiki/Kabsch_algorithm

    R=torch.ones(C.shape[0],3,3).cuda()
    for ib in range(C.shape[0]):



        U, S, V = torch.svd(C[ib,:,:])

        d = (torch.det(V) * torch.det(U.transpose(0,1))) < 0.0



        if d:
            S[-1] = -S[-1]
            V[-1, :] = -V[-1, :]

        E=torch.diag(torch.Tensor([1,1,1])).cuda()
        R[ib, :, :] = torch.mm(V, torch.mm(E, U.transpose(0, 1)))
    cors= torch.bmm(R, P.transpose(1, 2)).transpose(2, 1)


    return cors
